packages/byods/src/Errors/types.ts

```diff
@@ -0,0 +1,60 @@
+import {IMetaContext} from '../Logger/types';
+
+export type ErrorMessage = string;
+
+export enum ERROR_LAYER {
+  CALL_CONTROL = 'call_control',
+  MEDIA = 'media',
+}
+
+export enum ERROR_TYPE {
+  CALL_ERROR = 'call_error',
+  DEFAULT = 'default_error',
+  FORBIDDEN_ERROR = 'forbidden',
+  NOT_FOUND = 'not_found',
+  REGISTRATION_ERROR = 'registration_error',
+  SERVICE_UNAVAILABLE = 'service_unavailable',
+  TIMEOUT = 'timeout',
+  TOKEN_ERROR = 'token_error',
+  SERVER_ERROR = 'server_error',
+}
+
+export enum ERROR_CODE {
+  UNAUTHORIZED = 401,
+  FORBIDDEN = 403,
+  DEVICE_NOT_FOUND = 404,
+  INTERNAL_SERVER_ERROR = 500,
+  NOT_IMPLEMENTED = 501,
+  SERVICE_UNAVAILABLE = 503,
+  BAD_REQUEST = 400,
+  REQUEST_TIMEOUT = 408,
+  TOO_MANY_REQUESTS = 429,
+}
+
+export enum CALL_ERROR_CODE {
```

Comment:
Same over here... we can remove this and only keep errors pertaining to the current SDK in the code.

Reply:
@adhmenon Updated...!